src/View/View.php

```diff
@@ -830,7 +837,7 @@ public function extend($name)
                     if (!$parent) {
                         list($plugin, $name) = $this->pluginSplit($name);
                         $paths = $this->_paths($plugin);
-                        $defaultPath = $paths[0] . 'Element' . DIRECTORY_SEPARATOR;
+                        $defaultPath = $paths[0] . ucfirst(static::TYPE_ELEMENT) . DIRECTORY_SEPARATOR;
```

Comment:
Adding a function call adds overhead. Why not add a constant with the appropriate value?